Tests/Form/Type/ColorTypeTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Tests\Form\Type;
+
+use Sonata\CoreBundle\Form\Type\ColorType;
+
+class ColorTypeTest extends TypeTestCase
+{
+    public function testBuildForm()
+    {
+        // NEXT_MAJOR: Hack for php 5.3 only, remove it when requirement of PHP is >= 5.4
+        $that = $this;
+
+        $formBuilder = $this->getMockBuilder('Symfony\Component\Form\FormBuilder')->disableOriginalConstructor()->getMock();
+        $formBuilder
+            ->expects($this->any())
+            ->method('add')
+            ->will($this->returnCallback(function ($name, $type = null) use ($that) {
+                // NEXT_MAJOR: Remove this "if" (when requirement of Symfony is >= 2.8)
+                if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+                    if (null !== $type) {
+                        $isFQCN = class_exists($type);
+                        if (!$isFQCN && method_exists('Symfony\Component\Form\AbstractType', 'getName')) {
+                            // 2.8
+                            @trigger_error(
```

Comment:
@kunicmarko20 what was this about? It's a test...

Reply:
@greg0ire that part was just a copy/paste from other form type test classes, and I think that is the only reason I added it. If you check test for getParent or any other form type you will see the same stuff, maybe that parts should be removed from all tests?

Reply:
Probably... can you use `git log -S` to find the first commit that introduced that kind of stuff?

Reply:
https://github.com/sonata-project/SonataCoreBundle/commit/c62f7b47e5e9accaad43077cbecfbc37487619e1 
I think this is it.

Reply:
@ksom can you help? @Soullivaneuh you merged this :P

Reply:
Drunk to much :wine_glass: and :beers: this day I think. Maybe some :cocktail: to.